### PR TITLE
feat(ci): create workflow for gha stats

### PR DESCRIPTION
**Pull request by IvanShamatov**
[CI-490]

### Description

This is a workflow which is triggered by other workflow's completion event. It sends request to Anvil to fetch stats.

[CI-490]: https://toptal-core.atlassian.net/browse/CI-490

### CODE REVIEW

**Review by conf — APPROVED** [review]
LGTM, please, fix the failing checks and this is good to go.

File: .github/workflows/report_gha_stats_to_anvil.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: 'Report GHA stats to Anvil'
+on:
+  workflow_dispatch:
+  workflow_run:
+    workflows: ["Visual Testing", "Request review on PRs", "Consulting the Oracle", "Automatic Rebase"]
```
Comment (OleksandrNechai) [inline .github/workflows/report_gha_stats_to_anvil.yml:5]: What if tomorrow we add a new action and forget to add it to this list. What happens?

**Review by OleksandrNechai — APPROVED** [review]
LGTM, did not test.

File: .github/workflows/report_gha_stats_to_anvil.yml (hunk 1 above)
Comment (IvanShamatov) [reply .github/workflows/report_gha_stats_to_anvil.yml:5]: Not new action, but workflow, and yes, there is no solution for that right now unfortunately

**Pull request merged**